### PR TITLE
Landing Page Design Updation

### PR DESCRIPTION
**Pull request by VLBHAT31**
- Created only the landing page design
- Associated subpages and links are yet to be created.
Screenshot from 2024-03-27 23-37-15

### CODE REVIEW

File: webapp/landingpage/static/assets/demo/demo.js [hunk 1]
```diff
@@ -102,348 +102,7 @@ demo = {
...
-        }
-      }
-    };
-
-    var viewsChart = new Chart(e, a);
-  },
+  }, 
 
   initGoogleMaps: function() {
     var myLatlng = new google.maps.LatLng(40.748817, -73.985428);
```
Comment (kinderasteroid) [inline webapp/landingpage/static/assets/demo/demo.js:108]: why are we setting GoogleMaps in code?

**Review by kinderasteroid — COMMENTED** [review]
@VLBHAT31 When you Give PR. assign me in Reviewer Side okay. Else i wont Get Review notification and only thing i get is your comment

File: webapp/landingpage/static/assets/demo/demo.js (hunk 1 above)
Comment (VLBHAT31) [reply webapp/landingpage/static/assets/demo/demo.js:108]: no no that was bymistake left there which was in template  ill remove that part of code

Comment (kinderasteroid) [reply webapp/landingpage/static/assets/demo/demo.js:108]: Okay Remove and push your code to origin,  Enough

Comment (VLBHAT31) [reply webapp/landingpage/static/assets/demo/demo.js]: okay, pushed

**Review by kinderasteroid — APPROVED** [review]
Okay, Code Checked and is Approved from my end

**Comment by VLBHAT31** [comment]
@kinderasteroid should i write testcases for the code what ive written till now? or should i do other subpages and their connections from the sidebar?

**Comment by kinderasteroid** [comment]
> @kinderasteroid should i write testcases for the code what ive written till now? or should i do other subpages and their connections from the sidebar?

Like what Test cases?

**Comment by VLBHAT31** [comment]
U said testing is pending na so i thought like whether i have to write testcases

**Comment by VLBHAT31** [comment]
so ill start working on subpages then

**Comment by kinderasteroid** [comment]
@VLBHAT31 No No I Told UI Testing is pending from my End. You Can Proceed with your Work, Can you Update me your ETA? Because You need Also need our API's Right to Fetch Data From Logs and Authentication part? Will you work on connection or should i do it ?

**Comment by kinderasteroid** [comment]
> @VLBHAT31 No No I Told UI Testing is pending from my End. You Can Proceed with your Work, Can you Update me your ETA? Because You need Also need our API's Right to Fetch Data From Logs and Authentication part? Will you work on connection or should i do it ?
> 
> If you are working on it. i'll Provide you our APi and Its auth keys (if exists)

Update me with these Required Information

Static Badge

)

**Comment by VLBHAT31** [comment]
@kinderasteroid can you do the connections then? ill work on the subpage design by then

**Comment by kinderasteroid** [comment]
> @kinderasteroid can you do the connections then? ill work on the subpage design by then

Then You have to finish the work ASAP? Like By 30th Afternoon or something?

**Comment by VLBHAT31** [comment]
> > @kinderasteroid can you do the connections then? ill work on the subpage design by then
> 
> Then You have to finish the work ASAP? Like By 30th Afternoon or something?

Okay 30 afternoon is fine

**Comment by VLBHAT31** [comment]
- Subpages has been created 
- Connections to subpages from sidebar has been established 
- also the camera, notification, electric status upon click on main page redirects to that specific page as well.

**Comment by kinderasteroid** [comment]
@VLBHAT31 Can You Reflect the Changes i Requested In whatsapp, then we can add backend

**Review by kinderasteroid — APPROVED** [review]

**Pull request merged**